<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/Marvin-The-Martian-psd77710.png</normaloff>:/icons/Marvin-The-Martian-psd77710.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="listView">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Snap</enum>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="gridSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_Restart"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
